cookbook-react/src/recipes/radios/zecatDifapro/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import i18next from 'i18next';
+import clsx from 'clsx';
+
+import styles from './styles.module.scss';
+import ValidIcon from './assets/ic_valid.svg';
+import InvalidIcon from './assets/ic_invalid.svg';
+
+interface Props {
+  title: string;
+  info: string;
+  extraInfo: string;
+  disabled?: boolean;
+  name: string;
+  id: string;
+  inputStyle?: 'normal' | 'custom';
+  type?: 'radio' | 'checkbox';
+  checked: boolean;
+  onChange?: (event: React.FormEvent<HTMLInputElement>) => void;
+  wrapperClassName?: string;
+  titleClassName?: string;
+  infoClassName?: string;
+  extraInfoClassName?: string;
+}
+
+function ZecatDifapro({
+  title = i18next.t('ZecatDifapro:title'),
+  info = i18next.t('ZecatDifapro:info'),
+  extraInfo = i18next.t('ZecatDifapro:extraInfo'),
+  disabled = false,
+  name,
+  id,
+  inputStyle = 'normal',
+  type = 'radio',
+  checked = false,
+  onChange,
+  wrapperClassName,
+  titleClassName,
+  infoClassName,
+  extraInfoClassName
+}: Props) {
+  if (inputStyle === 'custom') {
+    return (
+      <div
+        className={clsx(
+          styles.wrapper,
+          `${wrapperClassName ?? ''}`,
+          { [styles.default]: !disabled, [styles.disabled]: disabled, [styles.selected]: checked },
+          'row middle'
+        )}
+        onChange={onChange}
+      >
+        <div className={`${styles.firstSection} row`}>
+          {checked ? (
+            <input
+              type="image"
+              src={ValidIcon}
+              alt={i18next.t('ZecatDifapro:selectedImg') as string}
+              name={name}
+              id={id}
+              className={styles.customIcon}
+              data-testid={`${name}-zecatdifapro`}
```

Comment:
Do not use test-id unless there's no other way to query the element. A good way to query inputs with react-testing-library is using `getByLabelText`:

```getByLabelText('ZecatDifapro:title')```

Just as a sidenote, in tests i18next does not work and the value it returns is the key you pass to the component